src/plugin.js

```diff
@@ -173,33 +175,34 @@ const onPlayerReady = (player, options) => {
         : <div class="vjs-errors-message">${player.localize(error.message)}</div>
         </div>`;
     }
-
     display = player.errorDisplay;
-
+    // The code snippet below is to make sure we dispose any child closeButtons before
+    // making the display closeable
+    if (display.getChild('closeButton')) {
+      display.removeChild('closeButton');
+    }
+    // Make the error display closeable, and we should get a close button
+    player.errorDisplay.closeable(true);
```

Comment:
Nevermind. So, if we're disposing the `display`, future references to `display` will be modifying the _old_ close button. I think we should re-assign `display` to the _new_ close button so we aren't modifying the old one below:

```js
display = player.getChild('errorDisplay');
```